src/hooks/useConnectWallet.ts

```diff
@@ -51,6 +59,9 @@ export const useConnectWallet = (): ConnectWalletAction => {
   const { disconnect } = useDisconnect()
   const { address, chain } = useAccount()
 
+  useSentrySetUser({ userAddress: state.address })
+  useSentrySetContextWallet({ userAddress: state.address })
+
```

Comment:
I don't it's a right place for this call. `useConnectWallet` could be called multiple times in different places, but we need to track an address once. We should call a hook on every page and it should look this:
```
useSentrySetUser() // this hook gets address and wallet name and propagates to sentry, no need to pass values to it
```